tests/test_ext_autodoc_configs.py

```diff
@@ -508,8 +521,9 @@ def test_autodoc_typehints_signature(app):
         '',
         '.. py:function:: missing_attr(c, a: str, b: Optional[str] = None) -> str',
         '   :module: target.typehints',
-        ''
+        '',
     ]
+    assert list(actual) == expected
```

Comment:
+0 for changing style. But all other cases should be updated also. Anyway, that should be another PR.

Reply:
This was leftover from a previous change which needed the separate variable. Will revert.